tests/provisioning-manifest-snapshot.yml

```
@@ -1 +1,5 @@
 - include: mvn:org.jahia.packages/jahia-additional-modules/JAHIA_VERSION/yaml/provisioning
+- installBundle:
+  - 'mvn:org.jahia.modules/sitemap/4.0.0-SNAPSHOT'
```

```suggestion
  - 'mvn:org.jahia.modules/sitemap
```
This could be better as it will get the LATEST snapshot (see here https://ops4j1.jira.com/wiki/spaces/paxurl/pages/3833866/Mvn+Protocol)